recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Support latest gtest
+    # https://github.com/msgpack/msgpack-c/commit/4a1db60af8d6bdb258a34352f5cd8d9800544372.patch
+    - gtest.patch
+    # This is needed for a bug in gcc version used where isnan is ambiguous. (::isnan and std::isnan)
+    - ambiguous_nan_inf.patch   # [linux]
```

Comment:
Still, ``msgpack_c.cpp`` is also including ``<vector>`` which is C++. I think the ambiguity derives from the ``using namespace std;`` statement on top, I am now thinking the better way would probably be to explicitly use ``std::isinf()`` in the code rather than relying on the ``using namespace std;`` statement.